cordformation/src/main/kotlin/net/corda/plugins/Node.kt

```diff
@@ -567,19 +598,61 @@ open class Node @Inject constructor(private val project: Project) {
         if (!config.hasPath("sshd.port")) {
             sshdPort(defaultSsh)
         }
-        val configDefaults = ConfigFactory.empty()
-            .withValue("dataSourceProperties.dataSource.url", ConfigValueFactory.fromAnyRef("jdbc:h2:file:./persistence/persistence;DB_CLOSE_ON_EXIT=FALSE;WRITE_DELAY=0;LOCK_TIMEOUT=10000"))
+
+        // TODO Refactor this code to extract the information from the config file
```

Comment:
Let's avoid leaving TODOs in the code. If it can be fixed now then let's do so. If it's a serious piece of work that needs a follow up task, then we'll have to raise a JIRA and scope it in.

Reply:
Removed